examples/using-wordpress/gatsby-node.js

```diff
@@ -1,108 +1,87 @@
+  const result = await graphql(`
+    {
+      allWordpressPage {
+        edges {
+          node {
+            id
+            slug
+            status
+            template
           }
         }
-      `
-    )
-      .then(result => {
-        if (result.errors) {
-          console.log(result.errors)
-          reject(result.errors)
+      }
+      allWordpressPost {
+        edges {
+          node {
+            id
+            slug
+            status
+            template
+            format
+          }
         }
+      }
+    }
+  `)
 
-        // Create Page pages.
-        const pageTemplate = path.resolve(`./src/templates/page.js`)
-        // We want to create a detailed page for each
-        // page node. We'll just use the Wordpress Slug for the slug.
-        // The Page ID is prefixed with 'PAGE_'
-        _.each(result.data.allWordpressPage.edges, edge => {
-          // Gatsby uses Redux to manage its internal state.
-          // Plugins and sites can use functions like "createPage"
-          // to interact with Gatsby.
-          createPage({
-            // Each page is required to have a `path` as well
-            // as a template component. The `context` is
-            // optional but is often necessary so the template
-            // can query data specific to each page.
-            path: `/${edge.node.slug}/`,
-            component: slash(pageTemplate),
-            context: {
-              id: edge.node.id,
-            },
-          })
-        })
-      })
-      // ==== END PAGES ====
+  // Check for any errors
+  if (result.errors) {
+    throw new Error(result.errors)
+  }
 
-      // ==== POSTS (WORDPRESS NATIVE AND ACF) ====
-      .then(() => {
-        graphql(
-          `
-            {
-              allWordpressPost {
-                edges {
-                  node {
-                    id
-                    slug
-                    status
-                    template
-                    format
-                  }
-                }
-              }
-            }
-          `
-        ).then(result => {
-          if (result.errors) {
-            console.log(result.errors)
-            reject(result.errors)
-          }
-          const postTemplate = path.resolve(`./src/templates/post.js`)
-          // We want to create a detailed page for each
-          // post node. We'll just use the Wordpress Slug for the slug.
-          // The Post ID is prefixed with 'POST_'
-          _.each(result.data.allWordpressPost.edges, edge => {
-            createPage({
-              path: edge.node.slug,
-              component: slash(postTemplate),
-              context: {
-                id: edge.node.id,
-              },
-            })
-          })
-          resolve()
-        })
-      })
-    // ==== END POSTS ====
+  // Access query results via object destructuring
+  const { allWordpressPage, allWordpressPost } = result.data
+
+  // Create Page pages.
```

Comment:
```suggestion
```

Not really needed--the code is sorta self-documenting here, right?

Reply:
@DylanTackoor this one is still here

Reply:
@wardpeet Whoops! Got it!